config/locales/server.en.yml

```diff
@@ -1503,9 +1503,9 @@ en:
     allow_user_locale: "Allow users to choose their own language interface preference"
     set_locale_from_accept_language_header: "set interface language for anonymous users from their web browser's language headers"
     support_mixed_text_direction: "Support mixed left-to-right and right-to-left text directions."
-    min_post_length: "Minimum allowed post length in characters"
-    min_first_post_length: "Minimum allowed first post (topic body) length in characters"
-    min_personal_message_post_length: "Minimum allowed post length in characters for messages"
+    min_post_length: "Minimum allowed post length in characters (excluding personal messages)"
```

Comment:
Is it just me or is the "in characters" a bit redundant? Not sure which ambiguity this would resolve. 😅 Also not there in `min_first_post_length`.

Reply:
yeah it is a bit redundant, agree we should clean up the setting names ... I guess it helps disambiguate between chars and words...